keyboards/naked64/info.json

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
This file doesn't render correctly:

![image](https://user-images.githubusercontent.com/18669334/67521199-3dc29880-f65f-11e9-93c2-dc76abe8703a.png)

Our KLE-to-info.json converter page doesn't give good values if you submit KLE input that includes keys that have been rotated. (This is due to how KLE positions rotated keys.)

Try replacing the file contents with this:

```json
{
    "keyboard_name": "Naked64", 
    "url": "https://salicylic-acid3.hatenablog.com/", 
    "maintainer": "Salicylic_acid3", 
    "width": 16,
    "height": 5,
    "layouts": {
        "LAYOUT": {
            "layout": [
                {"label":"PgUp", "x":0, "y":0},
                {"label":"Esc", "x":2, "y":0},
                {"label":"1", "x":3, "y":0},
                {"label":"2", "x":4, "y":0},
                {"label":"3", "x":5, "y":0},
                {"label":"4", "x":6, "y":0},
                {"label":"5", "x":7, "y":0},
                {"label":"6", "x":9, "y":0},
                {"label":"7", "x":10, "y":0},
                {"label":"8", "x":11, "y":0},
                {"label":"9", "x":12, "y":0},
                {"label":"0", "x":13, "y":0},
                {"label":"-", "x":14, "y":0},
                {"label":"Backspace", "x":15, "y":0},
                {"label":"PgDn", "x":0, "y":1},
                {"label":"Tab", "x":1.5, "y":1, "w":1.5},
                {"label":"Q", "x":3, "y":1},
                {"label":"W", "x":4, "y":1},
                {"label":"E", "x":5, "y":1},
                {"label":"R", "x":6, "y":1},
                {"label":"T", "x":7, "y":1},
                {"label":"Y", "x":9, "y":1},
                {"label":"U", "x":10, "y":1},
                {"label":"I", "x":11, "y":1},
                {"label":"O", "x":12, "y":1},
                {"label":"P", "x":13, "y":1},
                {"label":"{", "x":14, "y":1},
                {"label":"}", "x":15, "y":1},
                {"label":"Control", "x":1.25, "y":2, "w":1.75},
                {"label":"A", "x":3, "y":2},
                {"label":"S", "x":4, "y":2},
                {"label":"D", "x":5, "y":2},
                {"label":"F", "x":6, "y":2},
                {"label":"G", "x":7, "y":2},
                {"label":"H", "x":9, "y":2},
                {"label":"J", "x":10, "y":2},
                {"label":"K", "x":11, "y":2},
                {"label":"L", "x":12, "y":2},
                {"label":";", "x":13, "y":2},
                {"label":"'", "x":14, "y":2},
                {"label":"Enter", "x":15, "y":2},
                {"label":"Shift", "x":0.75, "y":3, "w":2.25},
                {"label":"Z", "x":3, "y":3},
                {"label":"X", "x":4, "y":3},
                {"label":"C", "x":5, "y":3},
                {"label":"V", "x":6, "y":3},
                {"label":"B", "x":7, "y":3},
                {"label":"N", "x":9, "y":3},
                {"label":"M", "x":10, "y":3},
                {"label":",", "x":11, "y":3},
                {"label":".", "x":12, "y":3},
                {"label":"/", "x":13, "y":3},
                {"label":"Up", "x":14, "y":3},
                {"label":"Shift", "x":15, "y":3},
                {"label":"Adjust", "x":1.75, "y":4, "w":1.5},
                {"label":"Alt", "x":3.5, "y":4, "w":1.5},
                {"label":"Space", "x":5, "y":4, "w":2},
                {"label":"Lower", "x":7, "y":4},
                {"label":"Raise", "x":9, "y":4},
                {"label":"Space", "x":10, "y":4, "w":2},
                {"label":"GUI", "x":12, "y":4},
                {"label":"Left", "x":13, "y":4},
                {"label":"Down", "x":14, "y":4},
                {"label":"Right", "x":15, "y":4}
            ]
        }
    }
}
```

This will look more keyboard-like.

![image](https://user-images.githubusercontent.com/18669334/67521613-17e9c380-f660-11e9-973f-18c2759281b0.png)

Reply:
Thank you. I fixed it.